<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionDetailFrame</class>
 <widget class="QFrame" name="QuestionDetailFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLabel" name="lb_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text_question">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>420</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tb_comment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>420</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btn_comment">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
    </font>
   </property>
   <property name="text">
    <string>回答</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
    </font>
   </property>
   <property name="text">
    <string>回答：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
